pom.xml

```diff
@@ -1067,6 +1075,54 @@
                 <version>4.5.1</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-avro-serializer</artifactId>
+                <version>${dep.confluent.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.kafka</groupId>
+                        <artifactId>kafka-clients</artifactId>
```

Comment:
Was thinking to keep the apache clients since people may not always use confluent. Also if other schema components to kafka not from confluent there might be better compatibility with the apache version. lmk what you think.